contrib/dex/README.md

```diff
@@ -0,0 +1,65 @@
+## Exposing DEX service
+After the cluster is up, you have to manually expose dex service using a ELB or Ingress.
```

Comment:
Maybe for now we could allow the user to specify the Service annotation `service.beta.kubernetes.io/aws-load-balancer-ssl-cert` which needs the ARN of the certificate?